website/source/docs/internals/discovery-chain.html.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: "docs"
+page_title: "Discovery Chain"
+sidebar_current: "docs-internals-discovery-chain"
+description: |-
+  The service discovery process can be modeled as a "discovery chain" which passes through three distinct stages: routing, splitting, and resolution. Each of these stages is controlled by a set of configuration entries.
+---
+
+# Discovery Chain <sup>beta</sup>
```

Comment:
if this ends up not being merged until GA we should make a note to fix this as we may have already made a 'beta' removal pass in the docs. Hopefully can be merged before though.

Reply:
We have not made a beta pass yet.

Reply:
Made an issue so we don't forget: https://github.com/hashicorp/consul/issues/6349